daemon/container.go

```diff
@@ -257,7 +257,7 @@ func (container *Container) Start() (err error) {
 			container.setError(err)
 			// if no one else has set it, make sure we don't leave it at zero
 			if container.ExitCode == 0 {
-				container.ExitCode = 128
+				container.ExitCode = 125
```

Comment:
Is exit code 128 ever defined to mean anything in  similar commands like chroot or bash?  Googling it shows a hole mismatch of definitions. I have no problem leaving it but we need to document what it means so the qe people could test for this exit code and describe what i